web/app/AttributeFormatter.js

```diff
@@ -34,6 +34,19 @@ Ext.define('Traccar.AttributeFormatter', {
         return Ext.getStore('DistanceUnits').formatValue(value, Traccar.app.getPreference('distanceUnit'));
     },
 
+    intervalFormatter: function (value) {
```

Comment:
I was thinking...
It is engine hours, why not it be formatted like just hours, floating-point hours. I do not think some body wants to know how much seconds his engine worked.
It will look like:
`5.04 h` or `189.26 h`

What do you think?